__tests__/app.test.js

```diff
@@ -0,0 +1,72 @@
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+const db = require("../db/connection");
+const app = require("../app");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api", () => {
+  describe("GET /topics", () => {
```

Comment:
There should be some error handling here too - what errors could you add for this?